elastic/kibana
# [Fleet] added logic to sanitize tags on UI

## Summary

Fixes https://github.com/elastic/kibana/issues/135775

Added logic to filter out special characters from tag name when creating/renaming on UI. Allowing letters, numbers, space, hyphen, underscore only. 
Limiting tag name to 20 characters.

To verify:
- Go to agent action Add / remove tags, type a new name
- Try entering special chars e.g. "{}|';: 
- Verify that the input doesn't allow entering these characters
- Verify that the input allows max 20 characters for tag name
- The same behavior is there for tag rename functionality

Pending: the same logic should be added on API or an error to be thrown if a tag contains special chars.


### Checklist
